README.md

```diff
@@ -521,6 +521,35 @@ metadata:
 type: kubernetes.io/tls
 ```
 
+## Post Factory Installation Updates
+
+In cases the starlingx system was already deployed once by the Deployment
+Manager with generic configurations, additional customized configurations are
+expected to be applied once again to finalize the configuration of the system.
+The Deployment Manager can read/update a configmap and orchestrate the
+configuration process accordingly.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+	name: factory-install
+	namespace: deployment
+data:
+	factory-installed: "true"  # Initial state (change to "true" if factory-installed)
+	factory-config-finalized: "false" # Optional (change to "true" if configuration is complete)
+	system-abcd-default-updated: "false" # Optional (change to "true" if resource default updated)
```

Comment:
Can we just use `system-default-update` instead?
I believe every DM deployment handles only one system.

Reply:
Would prefer to keep the method generic for all the resources, we need it for the systems and hosts currently though. The system name should expected to be correctly set, otherwise the DM is not good to handle multiple system resources in anyways.